gateway/cli/src/lightning_commands.rs

```diff
@@ -69,6 +71,25 @@ pub enum LightningCommands {
     },
     /// List active channels.
     ListActiveChannels,
+    /// Withdraw funds from the lightning node's on-chain wallet to a specified
+    /// address.
+    WithdrawOnchain {
+        /// The address to withdraw funds to.
+        #[clap(long)]
+        address: bitcoin::Address<NetworkUnchecked>,
+
+        /// The amount to withdraw, in satoshis.
+        #[clap(long)]
+        amount_sats: Option<u64>,
```

Comment:
Could we use an amount type here? Maybe even `AmountOrAll` so users can specify `all`

Reply:
Done.